pkg/podmanager/pod_lifecycle_manager.go

```diff
@@ -162,13 +162,14 @@ func (manager StandardPodLifecycleManager) UpdatePods(ctx context.Context, r cli
 
 // UpdateImageVersion updates a Pod container's image.
 func (manager StandardPodLifecycleManager) UpdateImageVersion(ctx context.Context, r client.Client, _ *fdbv1beta2.FoundationDBCluster, pod *corev1.Pod, containerIndex int, image string) error {
+	originalPod := pod.DeepCopy()
 	pod.Spec.Containers[containerIndex].Image = image
-	return r.Update(ctx, pod)
+	return r.Patch(ctx, pod, client.MergeFrom(originalPod))
 }
 
 // UpdateMetadata updates an Pod's metadata.
-func (manager StandardPodLifecycleManager) UpdateMetadata(ctx context.Context, r client.Client, _ *fdbv1beta2.FoundationDBCluster, pod *corev1.Pod) error {
-	return r.Update(ctx, pod)
+func (manager StandardPodLifecycleManager) UpdateMetadata(ctx context.Context, r client.Client, _ *fdbv1beta2.FoundationDBCluster, pod *corev1.Pod, *corev1.Pod) error {
```

Comment:
```suggestion
func (manager StandardPodLifecycleManager) UpdateMetadata(ctx context.Context, r client.Client, _ *fdbv1beta2.FoundationDBCluster, pod *corev1.Pod, originalPod *corev1.Pod) error {
```

Thanks for the PR, we probably want to move this behind a flag to allow users to enable the usage of patch or still make use of the update mechanism.